requirements_test.txt

```diff
@@ -1,3 +1,4 @@
 coverage>=4.0,<4.4  # TODO: https://github.com/benoitc/gunicorn/issues/1548
+mock; python_version < "3"
```

Comment:
This is already handled in `setup.py`.

Reply:
I think this form is cleaner, but maybe we do that for mock and unittest2 in a separate change.

Reply:
ok. removed this commit from pr